pyrobosim/pyrobosim/core/objects.py

```diff
@@ -35,6 +36,22 @@ def set_metadata(cls, filename):
         """
         cls.metadata = EntityMetadata(filename)
 
+    @classmethod
+    def add_metadata(cls, filename):
```

Comment:
see -- I like the style of this file, where `set_metadata()` was kept and `add_metadata()` was simply added.